src/pages/add-lock-exception/AddLockException.js

```diff
@@ -0,0 +1,146 @@
+import { useDataQuery, useDataMutation, useAlert } from '@dhis2/app-runtime'
+import i18n from '@dhis2/d2-i18n'
+import {
+    Card,
+    CenteredContent,
+    NoticeBox,
+    Button,
+    CircularLoader,
+} from '@dhis2/ui'
+import React, { useState } from 'react'
+import { useHistory } from 'react-router-dom'
+import LockExceptionsSubpageHeader from '../../components/LockExceptionsSubpageHeader/LockExceptionsSubpageHeader'
+import styles from './AddLockException.module.css'
+import AddLockExceptionForm from './AddLockExceptionForm/AddLockExceptionForm'
+
+const query = {
+    levels: {
+        resource: 'organisationUnitLevels',
+        params: {
+            fields: 'id,level,displayName',
+            paging: false,
+            order: 'level:asc',
+        },
+    },
+    groups: {
+        resource: 'organisationUnitGroups',
+        params: {
+            fields: 'id,displayName',
+            paging: false,
+        },
+    },
+    dataSets: {
+        resource: 'dataSets',
+        params: {
+            fields: 'id,displayName,periodType',
+            paging: false,
+        },
+    },
+}
+
+const mutation = {
+    resource: 'lockExceptions',
+    type: 'create',
+    params: params => params,
+}
+
+const AddLockException = () => {
```

Comment:
This component was [previously a modal rendered by the lock exceptions page component](https://github.com/dhis2/data-administration-app/blob/master/src/pages/lock-exception/LockException.js#L416-L428).

Extracting it into its own page improves not only the UX (the `material-ui` modal used to render this component did not make full use of the user's screen) but also aids maintenance by greatly reducing the complexity of the lock exceptions page component.